shared/js/media/video_player.js

```diff
@@ -237,6 +243,9 @@ function VideoPlayer(container) {
   // Set the video duration when we get metadata
   player.onloadedmetadata = function() {
     durationText.textContent = formatTime(player.duration);
+    slider.setAttribute('aria-valuemax', player.duration);
+    navigator.mozL10n.setAttributes(slider, 'playbackSeekBar',
+                                    {'duration': formatTime(player.duration)});
```

Comment:
define a variable here for the formatted duration so you only have to call formatTime() once in this function